tripper/utils.py

```diff
@@ -128,10 +128,22 @@ def parse_literal(literal: "Any") -> "Literal":
     if isinstance(literal, Literal):
         return literal
 
+    if hasattr(literal, "lang"):
+        lang = literal.lang
+    elif hasattr(literal, "language"):
+        lang = literal.language
+
+    if not lang and hasattr(literal, "datatype"):
+        datatype = (
+            literal.datatype
+            if literal.__class__ == str
+            else str(literal.datatype)
+        )
+
     # This will handle rdflib literals correctly and probably most other
     # literal representations as well.
     if hasattr(literal, "value"):
-        return Literal(literal.value)
+        return Literal(literal.value, lang=lang, datatype=datatype)
```

Comment:
The rdflib documentation states that `Literal` contains a literal value and either a datatype or language tag (i.e. not both). [Link](https://rdflib.readthedocs.io/en/stable/rdf_terms.html) (see the first paragraph)

Is it still correct to supply both?

Reply:
`tripper.Literal` and `rdflib.Literal` are two different things. Like rdflib, tripper does not expect language and datatype to be provided at the same time, so typically either `lang` or `datatype` is None.